services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> Movie:
```

Comment:
The function `get_movies` is expected to return a QuerySet (a list of movies), but the return type hint suggests that it will return a single Movie instance. This is misleading and can cause confusion. Change `-> Movie:` to `-> QuerySet:` to reflect the actual return type.